boards/arm/lpcxpresso55s69/lpcxpresso55s69.dtsi

```diff
@@ -90,6 +90,64 @@
 	current-speed = <115200>;
 };
 
+&flash0 {
+	/*
+	 * For more information, see:
+	 * https://docs.zephyrproject.org/latest/guides/dts/legacy-macros.html#legacy-flash-partitions
+	 */
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		boot_partition: partition@0 {
+			label = "mcuboot";
+			reg = <0x00000000 0x00010000>;
+			read-only;
+		};
+		/*
+		 * The flash starting at 0x10010000 and ending at
+		 * 0x1001ffff (sectors 16-31) is reserved for use
+		 * by the application.
+		 */
+		storage_partition: partition@1e000 {
+			label = "storage";
+			reg = <0x0001e000 0x00002000>;
+		};
+		slot0_partition: partition@20000 {
+			label = "image-0";
+			reg = <0x00020000 0x00020000>;
+		};
+		slot1_partition: partition@40000 {
+			label = "image-1";
+			reg = <0x00040000 0x00020000>;
+		};
+		scratch_partition: partition@60000 {
+			label = "image-scratch";
+			reg = <0x00060000 0x0003d800>;
```

Comment:
There isn't much point in making scratch this large. We have a rather large flash device, it would be better to make the two slots somewhat less than half, and make the scratch a few sectors at most (there are wear issues if scratch is too small). Realistically, we should be configuring `BOOT_SWAP_USING_MOVE` and not have a scratch partition. This requires that slot 0 be only a single sector larger.